suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -262,7 +267,9 @@ export const createDescriptorPreloadedDiscoveryThunk = createThunk(
 
         if (!deviceAccessResponse.success) return false;
 
-        const discoveryNetworksTotalCount = filterUnavailableNetworks(networks, device).length;
+        const discoveryNetworksTotalCount = filterBlacklistedNetworks(
```

Comment:
maybe create some function (in this file only) like this:
```tsx
const getAvailableNetworks = flow(filterBlacklistedNetworks, filterUnavailableNetworks)
...
const discoveryNetworksTotalCount = getAvaiableNetworks(network, device).length
```

This should avoid some duplicities, and can be used also lower

Reply:
[d322446](https://github.com/trezor/trezor-suite/pull/10241/commits/d3224461c37cbe1e69ccf60b6b83ee37767c1c1b)